### ISSUE TITLE
Slackの自動招待フォームのURLを支部ページに掲載する

### ISSUE THREAD

**Issue by aibax** [jaws-ug-kanazawa/jaws-ug-kanazawa.github.io#28]
JAWS-UG 金沢 Slack 申込フォーム
https://docs.google.com/forms/d/e/1FAIpQLSd1HAli2lS-_qzASY27HhpHkOnaMiYPKNdyaJ_eL6uVT63xng/viewform

支部ページ
https://jaws-ug.jp/bs/kanazawa/

**Comment by aibax** [comment]
URLをbit.lyで短縮
http://bit.ly/jawsug-kanazawa-slack

**Comment by aibax** [comment]
掲載完了
https://jaws-ug.jp/bs/kanazawa/
